src/modules/storage/LedgerStorage.ts

```diff
@@ -1130,7 +1130,7 @@ export class LedgerStorage extends Storages
                 SELECT
                     TX.address,
                     TX.block_height as height,
-                    STRFTIME('%s', '2020-01-01 00:00:00') * 1000 + TX.block_height * 10 * 60000 as block_time,
+                    TX.time_stamp as block_time,
```

Comment:
`unlock_time` also needs to be changed.
```
    (TX.time_stamp + (TX.unlock_height - TX.block_height) * 10 * 60) as unlock_time,
```

Reply:
https://github.com/bpfkorea/stoa/issues/252